docs/dev/installer.md

```diff
@@ -0,0 +1,188 @@
+
+For example, this allows the `installer` script to ensure that the deployed Dashboard and the RBAC permissions are consistent.
+
+## Help command
+
+The `help` command shows the supported commands and options by the script:
+
+```bash
+./scripts/installer help
+```
+
+It should output the following message:
+
+```bash
+Global command syntax:
+        installer COMMAND [OPTIONS]
+
+Accepted commands:
+        help|h                                  Prints this help
+        install|i                               Installs the dashboard
+        uninstall|u                             Uninstalls the dashboard
+        build|b                                 Builds the manifests and docker image and outputs manfiests in the console
+
+Accepted options:
+        [--debug]                               Prints additional messages in the console
+        [--csrf-secure-cookie]                  Enable secure CSRF cookie
+        [--openshift]                           Will build manifests for openshift
+        [--read-only]                           Will build manifests for a readonly deployment
+        [--namespace <namespace>]               Will override install namespace
+        [--pipelines-namespace <namespace>]     Override the namespace where Tekton Pipelines is installed (defaults to tekton-pipelines)
+        [--triggers-namespace <namespace>]      Override the namespace where Tekton Triggers is installed (defaults to tekton-pipelines)
+        [--ingress-url <url>]                   Will create an additional ingress with the specified url
+        [--ingress-secret <secret>]             Will add ssl support to the ingress
+```
+
+## Install command
+
+The `install` command is used to install the Tekton Dashboard in a cluster. It supports both Kubernetes and OpenShift.
+
+Note that OpenShift supports two install methods:
+- Tekton Pipelines and Triggers installed by openshift pipelines operator
+- Tekton Pipelines and Triggers installed manually using YAML manifests
+
+Examples below illustrate the main `install` options:
+- [Installing on Kubernetes](#installing-on-kubernetes)
+- [OpenShift with Tekton Pipelines and Triggers installed by OpenShift Pipelines Operator](#openshift-with-tekton-pipelines-and-triggers-installed-by-openshift-pipelines-operator)
+- [OpenShift with Tekton Pipelines and Triggers installed manually using YAML manifests](#openshift-with-tekton-pipelines-and-triggers-installed-manually-using-yaml-manifests)
+- [Installing in a custom namespace](#installing-in-a-custom-namespace)
+- [Install ingress](#install-ingress)
+
+### Installing on Kubernetes
+
+To install the Tekton Dashboard on Kubernetes, using the default options, run the command:
+
+```bash
+./scripts/installer install
+```
+
+This will install the Dashboard in the `tekton-pipelines` namespace, assuming Tekton Pipelines and Triggers are installed in the `tekton-pipelines` namespace.
```

Comment:
```suggestion
This will install the Dashboard in the `tekton-pipelines` namespace and assumes that Tekton Pipelines and Triggers are also installed in the `tekton-pipelines` namespace.
```